packages/edit-site/src/components/global-styles/font-size.js

```diff
@@ -0,0 +1,239 @@
+		} else {
+			// If custom fluid values are disabled, set fluid to true.
+			updateFontSize( 'fluid', true );
+		}
+	};
+
+	const handleMinChange = ( value ) => {
+		// If the user is resetting the value, use the initial value.
+		const newValue = value ?? fontSize.size;
+		updateFontSize( 'fluid', { ...fontSize.fluid, min: newValue } );
+	};
+
+	const handleMaxChange = ( value ) => {
+		// If the user is resetting the value, use the initial value.
+		const newValue = value ?? fontSize.size;
+		updateFontSize( 'fluid', { ...fontSize.fluid, max: newValue } );
+	};
+
+	const updateFontSize = ( key, value ) => {
+		const newFontSizes = sizes.map( ( size ) => {
+			if ( size.slug === slug ) {
+				return { ...size, [ key ]: value }; // Create a new object with updated key
+			}
+			return size;
+		} );
+
+		setFontSizes( {
+			...fontSizes,
+			theme: newFontSizes,
+		} );
+	};
+
+	const handleRemoveFontSize = () => {
+		// Navigate to the font sizes list.
+		goBack();
+
+		const newFontSizes = sizes.filter( ( size ) => size.slug !== slug );
+		setFontSizes( {
+			...fontSizes,
+			theme: newFontSizes,
+		} );
+	};
+
+	return (
+		<VStack spacing={ 4 }>
+			<HStack justify="space-between" align="flex-start">
+				<ScreenHeader
+					title={ fontSize.name }
+					description={ sprintf(
+						/* translators: %s: font size preset name. */
+						__( 'Manage the font size %s.' ),
+						fontSize.name
+					) }
+				/>
+				<FlexItem>
+					<Spacer marginTop={ 3 } marginBottom={ 0 } paddingX={ 4 }>
+						<DropdownMenu
+							trigger={
+								<Button
+									size="small"
+									icon={ moreVertical }
+									label={ __( 'Menu' ) }
+								/>
+							}
+						>
+							<DropdownMenuItem onClick={ handleRemoveFontSize }>
+								<DropdownMenuItemLabel>
+									{ __( 'Delete' ) }
+								</DropdownMenuItemLabel>
+							</DropdownMenuItem>
+						</DropdownMenu>
+					</Spacer>
+				</FlexItem>
+			</HStack>
+
+			<View>
+				<Spacer paddingX={ 4 }>
+					<VStack spacing={ 4 }>
+						<FlexItem>
+							<FontSizePreview fontSize={ fontSize } />
+						</FlexItem>
+
+						<InputControl
+							label={ __( 'Name' ) }
+							value={ fontSize.name }
+							onChange={ handleNameChange }
+						/>
+
+						<SizeControl
+							label={ __( 'Size' ) }
+							value={ fontSize.size }
+							withSlider
+							onChange={ handleFontSizeChange }
+							hasUnits={ hasUnits }
+							withReset={ false }
+						/>
+
+						<ToggleControl
+							label={ __( 'Fluid typography' ) }
+							checked={ isFluid }
+							onChange={ handleFluidChange }
+						/>
+
+						{ isFluid && (
+							<ToggleControl
+								label={ __( 'Custom fluid values' ) }
+								help={ __(
+									'Set custom min and max values for the fluid font size.'
+								) }
+								checked={ isCustomFluid }
+								onChange={ handleCustomFluidValues }
+							/>
+						) }
+
+						{ isCustomFluid && (
+							<>
+								<SizeControl
+									label={ __( 'Minimun' ) }
```

Comment:
@jasmussen I'd appreciate your input about the wording of this label.

Reply:
Assuming you're referring to this part:

![Screenshot 2024-06-20 at 12 32 49](https://github.com/WordPress/gutenberg/assets/1204802/3071bd6a-ea0c-4f24-9235-ad8f60dd9257)

Let's go with the full labels, "Minimum" and "Maximum", mostly because in other places of the UI, like "Minimum height" for the Cover block, we also use the full word. The help text reads okay as is, to me, but if need be that can be more verbose too.

Do note, though, that there's a typo at present. They say "Minimu**n**", and "Maximu**n**", replace the n with m.

Separate to this PR, we should look at updating the default font size control to be the correct 40px hight and use the new unit style. The Image block looks correct, note the unit selector and input height, as well as inner padding:

![Screenshot 2024-06-20 at 12 35 10](https://github.com/WordPress/gutenberg/assets/1204802/e2836f44-23e9-42ed-8eed-a4bd2d9f4c22)

Then compare to the controls here:

![Screenshot 2024-06-20 at 12 32 29](https://github.com/WordPress/gutenberg/assets/1204802/5d5585b5-759b-4df0-b7c9-1d24f7c8cd68)

Not specifically your task, but we want to update all of these instances so they have the proper 40px defaults.

Reply:
Yes, we were referring to that part. Ok, let's stick to 'Minimum' and 'Maximum'. Thanks for noticing the misspelled words. Fixed them here: [`0c8ecde` (#62328)](https://github.com/WordPress/gutenberg/pull/62328/commits/0c8ecdeb071f2167e1f8e73d1800205b1157a8ff)